homeassistant/components/generic/camera.py

```diff
@@ -157,6 +196,11 @@ def name(self):
         """Return the name of this device."""
         return self._name
 
+    @property
+    def unique_id(self) -> str:
+        """Return a unique ID."""
```

Comment:
Set `self._attr_unique_id` in `__init__` instead to simplify the code

Reply:
Thanks! Done